test/unit/test_asyncify.py

```diff
@@ -50,3 +51,16 @@ def test_asyncify_blacklist_and_whitelist(self):
         proc = run_process(WASM_OPT + [self.input_path('asyncify-pure.wast'), '--asyncify', '--pass-arg=asyncify-whitelist@main', '--pass-arg=asyncify-blacklist@main'], stdout=subprocess.PIPE, stderr=subprocess.STDOUT, check=False)
         self.assertNotEqual(proc.returncode, 0, 'must error on using both lists at once')
         self.assertIn('It makes no sense to use both a blacklist and a whitelist with asyncify', proc.stdout)
+
+    def test_asyncify_imports(self):
+        def test(args):
+            return run_process(WASM_OPT + [self.input_path('asyncify-sleep.wast'), '--asyncify', '--print'] + args, stdout=subprocess.PIPE).stdout
+
+        normal = test(['--pass-arg=asyncify-imports@env.sleep'])
+        temp = tempfile.NamedTemporaryFile().name
+        with open(temp, 'w') as f:
+            f.write('env.sleep')
+        response = test(['--pass-arg=asyncify-imports@@%s' % temp])
```

Comment:
Is the first @ a general delimiter for the argument passing mechanism and the second a part of the response file mechanism? That seems rather confusing, but probably not worth doing anything about. Maybe you can call it out in the comment in Asyncify.cpp so people don't think it's a typo or something?

Reply:
Thanks, good idea, added.